tools/report-converter/codechecker_report_converter/analyzers/sanitizers/ub/parser.py

```diff
@@ -41,6 +41,117 @@ class Parser(SANParser):
         # Checker message.
         r'(?P<message>[\S \t]+)')
 
+    # This list of diagnostic messages is extracted from the LLVM project:
+    #   https://github.com/llvm/llvm-project
+    # /llvm/llvm-project/blob/main/compiler-rt/lib/ubsan/ubsan_handlers.cpp
```

Comment:
the `/llvm/llvm-project/` prefix is reduntant.

Reply:
Please change this **not** to refer to `/blob/main` or something that is a constantly changing resource, but something that is fixed. If you press <kbd>Y</kbd> while browsing a file on GitHub, the browser's address bar will automatically transform into a link to the currently viewed commit, whose hash should not change, so is a good reference point which version of the project we investigated.